### PR TITLE
Enable session trace

### PR DESCRIPTION
**Pull request by abhinab-yb**
```
yugabyte=# set yb_debug_log_docdb_requests to 1;
SET
yugabyte=# create table users(id int);
CREATE TABLE
```

```
0220 16:41:28.982223 (+    99us) pg_session.cc:189] Apply
0220 16:41:28.982301 (+    78us) pg_session.cc:189] Apply
0220 16:41:28.982380 (+    79us) pg_session.cc:189] Apply
0220 16:41:28.982476 (+    96us) pg_session.cc:189] Apply
0220 16:41:28.983306 (+   830us) pg_session.cc:189] Apply
0220 16:41:28.984041 (+   735us) pg_session.cc:189] Apply
0220 16:41:28.984205 (+   164us) pg_session.cc:546] FlushOperations
0220 16:41:28.995754 (+ 11549us) pg_session.cc:216] Apply
0220 16:41:28.998488 (+  2734us) pg_session.cc:216] Apply
I0220 16:41:29.000913 21464448 ybccmds.c:508] Creating Table yugabyte.public.users

```

### CODE REVIEW

File: src/yb/yql/pggate/pg_session.cc [hunk 1]
```diff
@@ -319,12 +322,17 @@ Status PgSession::IsDatabaseColocated(const PgOid database_oid, bool *colocated,
...
+  TRACE_TO(trace_, __func__, "testing trace $0 $1", 9, 8);
+
   tserver::PgDropDatabaseRequestPB req;
   req.set_database_name(database_name);
   req.set_database_oid(database_oid);
 
   RETURN_NOT_OK(pg_client_.DropDatabase(&req, CoarseTimePoint()));
   RETURN_NOT_OK(DeleteDBSequences(database_oid));
+
+  LOG(INFO) << (trace_ ? trace_->DumpToString(true) : "Not collected");
```
Comment (vrajat) [inline src/yb/yql/pggate/pg_session.cc:334]: Is a trace only dumped if database is dropped?

**Review by vrajat — COMMENTED** [review]
Can you paste an example in PR description?

File: src/yb/yql/pggate/pg_session.cc (hunk 1 above)
Comment (abhinab-yb) [reply src/yb/yql/pggate/pg_session.cc:334]: yeah, I tested the dump using that

Comment (vrajat) [reply src/yb/yql/pggate/pg_session.cc:334]: Can you dump at the end of a query?

Comment (abhinab-yb) [reply src/yb/yql/pggate/pg_session.cc]: done

**Review by vrajat — APPROVED** [review]

**Pull request merged**